scripts/copy_qubit.py

```diff
@@ -72,9 +72,16 @@ def get_data(file_path):
                 read=True
             elif(read and row[sample_index]):
                 #this is every other row
-                data[row[sample_index]]={}
-                data[row[sample_index]]['concentration']=row[conc_index]
-                data[row[sample_index]]['unit']=row[unit_index]
+                if row[sample_index] in data:
+                    #Sample is duplicated, drop the key
+                    log.append("sample {0} has two rows in the Qubit CSV file. Please check the file manually.".format(row[sample_index]))
+                    del data[row[sample_index]]
```

Comment:
So you don't want a entry in `data` at all, for samples with duplicate row ? In that case wont a sample appear again in next iteration if it have occurred 3 (or odd numbered) times (if that is possible).